core/vm/interface.go

```diff
@@ -35,6 +35,8 @@ type StateDB interface {
 	SetNonce(common.Address, uint64)
 
 	GetCodeHash(common.Address) common.Hash
+	GetProof(common.Address) ([][]byte, error)
```

Comment:
Sorry for more nitpicks - but now that we have a type, it's better to return the `ProofList` type rather than the `[][]byte`.

Reply:
ok, now it return the ProofList, which is defined in the vm-interface.